moped-etl/prefect/moped_test_create_flow.py

```diff
@@ -8,42 +8,54 @@
 Labels: TBD
 """
 
-import argparse
-parser = argparse.ArgumentParser(description='Prefect flow for Moped Editor Test Instance Deployment')
-parser.add_argument('-m', '--mike', help='Run Mike\'s tasks', action='store_true')
-parser.add_argument('-f', '--frank', help='Run Frank\'s tasks', action='store_true')
-args = parser.parse_args()
-print(args)
+# import python standard library packages
+import os
 
-from venv import create
-import json
-import boto3
+# import pypi packages
 import prefect
-import sys, os
-import subprocess
-
+from prefect.run_configs import UniversalRun
 
-# Prefect
-from prefect import Flow, task
 
 import psycopg2
+
+# import package components
+from prefect import Flow, task
 from psycopg2.extensions import ISOLATION_LEVEL_AUTOCOMMIT
 
+from tasks.ecs import *
+
+
+# Import and setup argparse.
+# This is intended to aid development and will be removed prior to PRing torward main.
+import argparse
+
+parser = argparse.ArgumentParser(
+    description="Prefect flow for Moped Editor Test Instance Deployment"
+)
+parser.add_argument("-m", "--mike", help="Run Mike's tasks", action="store_true")
+parser.add_argument("-f", "--frank", help="Run Frank's tasks", action="store_true")
+parser.add_argument("-p", "--provision", help="Provision", action="store_true")
+parser.add_argument("-d", "--decomission", help="Decomission", action="store_true")
+args = parser.parse_args()
```

Comment:
@mddilley wrote:
> When I was finishing out the API/DB code this morning, I was able to get by without these args. If you are using it, I'm happy to keep it around, but I wanted to let you know in case you were keeping it around for me.

I think, based on our prefect-centered discussion the other day that this isn't very prefect-like, and should probably go. I propose that I strike them in lieu of registering multiple flows and running them from the prefect cloud UI for the time being if that's suitable to you. Lots of ways to do it.